Resources/Prototypes/Traits/physical.yml

```
@@ -578,12 +578,21 @@
+              Slash: -0.3
+              Heat: -0.3
+              Cold: -0.3
+              Shock: -0.3
+              Caustic: -0.3
+              Asphyxiation: -0.25
+              Bloodloss: -0.25
+              Poison: -0.3
+              Radiation: -0.3
+              Genetic: -0.3
+        - type: BloodDeficiency # Incredibly actually works
+          bloodLossPercentage: -0.025            
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)